test/matchers/shared_examples/has_valid_rspec_message_methods.rb

```diff
@@ -7,6 +7,7 @@ module HasValidRspecMessageMethods
         case validator_sym
         when :aspect_ratio then matcher.rejecting(:square)
         when :attached then matcher
+        when :limit then matcher.min(1).max(5)
```

Comment:
I think, like other `when` around, that it should be a wrong `min` and/or `max` to test the Rspec failure message?